docs/Makefile

```diff
@@ -223,3 +224,7 @@ dummy:
 	$(SPHINXBUILD) -b dummy $(ALLSPHINXOPTS) $(BUILDDIR)/dummy
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
+
+.PHONY: serve
+serve:
+	sphinx-autobuild -b html $(ALLSPINXOPTS) --watch ../i3ipc . $(BUILDDIR)/html
```

Comment:
Well it's kind of a tricky dependency because it's not in my package manager (Fedora). But it's the easiest way to get the file watcher working so I think it's worth it.

I think the best solution is to add a check for it in the PATH and give installation instructions when it's not there.